Pages/content.html

```diff
@@ -62,6 +62,7 @@
 <h3>Access Content files here.</h3>
 <!- Sharing content. -->
 <h4>E-Books:</h4><a href="../Content/intro.pdf" target="_blank">Introduction</a>
+<h4>Videos:</h4><a href="../Content/Video/meher_baba.mp4" target="_blank"" >Video</a>
```

Comment:
Please rename the _hyperlink_ to something like what is the video about. 
**Example**: Message to West

```suggestion
<h4>Videos:</h4><a href="../Content/Video/meher_baba.mp4" target="_blank"" >Message to West</a>
```